### ISSUE TITLE
[BUG] - App is not listed under VPNs in settings and not asking for VPN permissions

### ISSUE THREAD

**Issue by user12257** [zaneschepke/wgtunnel#390]
**Describe the bug**
The App is not asking me for permission to controll the VPN Settings. I cannot connect to any vpn, since the App does not have the permission. Under Settings/VPN it also does not appear. 

**Smartphone (please complete the following information):**

- Device: [Pixel 8 Pro]
- Android Version: [ Android 14]
- App Version [3.5.2]
- Backend: [Userspace]

**To Reproduce**

1. Download App from Github
2. Install
3. try to connect to any vpn or change the rights in settings.

**Expected behavior**
Being able to give the App the permission to change the VPN settings

**Screenshots (Only if necessary)**

**Additional context**

**Comment by zaneschepke** [comment]
Hello! The app should ask for VPN permission when you first try to turn on a tunnel. 

Is that not working for you?

Is this the official release version or the nightly version?

**Comment by user12257** [comment]
Yeah, requesting permission when turning on a VPN was not working. I realized now that the reason was, that I had another vpn setup to always connect. I assumed, that I would still be able to see the other VPN apps in the VPN Settings and give them permission. But apparently I needed to turn off my other VPN to set up the permissions. 

I am new to android, so I needed a while to realize that :D Thank you for your quick reply though
